bgpd/bgp_vty.c

```diff
@@ -8589,6 +8589,7 @@ DEFPY (neighbor_soo,
 		peer_af_flag_unset(peer, afi, safi, PEER_FLAG_SOO);
 	}
 
+	ecommunity_free(&ecomm_soo);
```

Comment:
Why is this needed?

Reply:
fixed it, if above fail we need to free it.